.github/workflows/release.yml

```diff
@@ -2,20 +2,20 @@ name: Automated product release
 
 on:
   pull_request:
-    branches: [ master ]
-    types: [ closed ]
-
+    branches: [master]
+    types: [closed]
 
 jobs:
   check-release:
     name: Check release required
-    runs-on: ubuntu-latest
     if: ${{ github.event.pull_request.merged && endsWith(github.repository, '-private') != true }}
```

Comment:
@marcin-cebo this whole entry is a key-value object, so order actually doesn't matter and I changed it just to see from the start when it will run and only after that 'where' :)